app/src/components/main/paginationToolbar/noItemMessage/noItemMessage.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import { NoItemMessage as MainNoItemMessage } from '../../noItemMessage';
+
+const messages = defineMessages({
+  noResults: {
+    id: 'LogsGrid.noResults',
+    defaultMessage: 'No results found',
+  },
+});
+
+export const NoItemMessage = injectIntl(({ intl }) => (
```

Comment:
What's the purpose of this component?

You can use existing component in the following way:
```
import { NoItemMessage } from 'components/main/noItemMessage';
import { COMMON_LOCALE_KEYS } from 'NO_RESULTS';

// ...

<NoItemMessage message={formatIntl(COMMON_LOCALE_KEYS.NO_RESULTS)} />
```

Reply:
fixed